CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Welcome!
+Thank you for your interest in Dashing! This exceptionally handsome
+framework welcomes all input, but being stylish requires certain protocols be followed. :bowtie:
+
+Below you will find a set of guidelines that will ensure the best outcome for you _and_ Dashing.
+<a name="issues"></a>
+## Have an Issue
+If you run into problems with Dashing, please take these steps before submitting an issue:
+
+1. Check the [Troubleshooting Guide]() in the wiki.
+2. Use the [GitHub Issue Search](https://help.github.com/articles/searching-issues/) to check if the issue has already been reported.
+3. Submit your issue to the [best place](). Please provide as much helpful information as possible, preferably making use of a [reduced test case](https://www.google.ca/search?#q=reduced%20test%20case).
+
+**Support requests should be directed to [Stack Overflow](http://stackoverflow.com/questions/tagged/dashing).**
+<a name="features"></a>
+## Feature Requests
+Feature requests are welcome, but take a moment and consider whether your idea
+fits with the scope and aims of the project. A good rule of thumb is to apply the 80/20 rule: every feature should be useful to at least 80% of Dashing. Dashing has many use cases, and it's impossible to accommodate every edge case.
+
+If you feel that you have a really amazing, super neato idea that doesn't quite fit with the core use of Dashing, it may be a good candidate for an external Gem which supercharges a project. An excellent example of this is [dashing-contrib](https://github.com/QubitProducts/dashing-contrib).
```

Comment:
Mention that we should add them here: https://github.com/Shopify/dashing/wiki/Additional-Widgets#other-third-party-tools